apps/custom-ui-floating-window-demo/components/ContactList.js

```diff
@@ -0,0 +1,61 @@
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
+import { startOutgoingCall } from "../shared/socket";
+import Footer from "./Footer";
+
+export default function ContactList(context) {
+  const router = useRouter();
+  let [contacts, setContacts] = useState([]);
+
+  useEffect(() => {
+    // Get the contacts and list them
+    fetch("/api/getContacts")
+      .then((res) => res.json())
+      .then((data) => {
+        setContacts(data);
+      });
+  }, [router]);
+
+  return (
+    <div class="container-fluid">
```

Comment:
in React the correct class attribute is className
```suggestion
    <div className="container-fluid">
```

can you fix it across whole project? Probably eslint can help you fix it automatically, for example with this plugin https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unknown-property.md

Reply:
Good tip. Added the fixes and updated eslintrc